doc/source/reference/random/legacy.rst

```diff
@@ -39,7 +35,7 @@ are produced in pairs. It is important to use
    mt19937 = MT19937(12345)
    lg = RandomState(mt19937)
 
-   # Identical output
+   # Different output, sorry.
```

Comment:
Should this block be removed since it doesn't actually show what it was written to show?  If you really wanted to show it, you could use

```
mt19937 = MT19937()
mt19973.state = rs.get_state()
lg = RandomState(mt19937)
```

Reply:
Good point. That workaround is worth documenting.